packages/web/src/components/gcds-header/stories/overview.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import * as Header from './gcds-header.stories';
+
+<Meta of={Header} name="Overview" />
+
+# Header<br/>`<gcds-header>`
+
+_Also called: global header, brand header._
+
+The header is the responsive Government of Canada branded header landmark.
+
+<Canvas
+  of={Header.Default}
+  story={{ inline: true }}
+/>
+
+## Examples
+<br/>
+
+### Default
+
+<Canvas
+  of={Header.Default}
+  story={{ inline: true }}
+/>
+
+### Slots
```

Comment:
Could we maybe have an extra story here that shows all slots at once? Just so it's easier for the user to understand the order. For example, the menu slot is before the breadcrumbs slot, etc.